servicex/app/main.py

```diff
@@ -63,6 +63,46 @@ def show_version(show: bool):
         raise typer.Exit()
 
 
+def _display_results(out_dict):
```

Comment:
Is this something others might find useful? e.g. should this get moved into the central library?

Reply:
I was wondering that. For the moment I decided that `servicex.app` was that namespace...